### PR TITLE
Ensure all swri::Subscriber members are initialized

### PR DESCRIPTION
**Pull request by pjreed**
Some member variables in swri::Subscriber were uninitialized by default; this ensures all of them are initialized.

### CODE REVIEW

**Comment by pjreed** [comment]
As long as I'm in here, looking at subscriber_impl.h:73, it appears that if the stamp in a header is 0, this will very quickly cause an overflow of `total_latency_`.  Maybe `processHeader` should skip the latency check if  the header's stamp isn't set?

File: swri_roscpp/include/swri_roscpp/subscriber_impl.h [hunk 1]
```diff
@@ -75,7 +75,8 @@ class SubscriberImpl
       min_latency_ = latency;
       max_latency_ = latency;
       total_latency_ = latency;
-    } else {
+    }
+    else {
```
Comment (elliotjo) [inline swri_roscpp/include/swri_roscpp/subscriber_impl.h]: Recommend undoing these style changes to remain consistent.

Comment (pjreed) [reply swri_roscpp/include/swri_roscpp/subscriber_impl.h]: Any opinions on how that check should be handled if the header stamp isn't set?

**Comment by evenator** [comment]
LGTM

**Pull request merged**